create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusSourceType.java

```diff
@@ -0,0 +1,22 @@
+package no.sikt.nva.scopus;
+
+public enum ScopusSourceType {
+
+    JOURNAL("j");
+
+    public final String code;
+
+    ScopusSourceType(String code) {
+        this.code = code;
+    }
+
+    public static ScopusSourceType valueOfCode(String code) {
+        for (ScopusSourceType c : values()) {
+            if (c.code.equals(code)) {
+                return c;
+            }
+        }
```

Comment:
Probably go for:

`

        return Arrays.stream(values())
            .filter(value -> value.code.equals(code))
            .findAny()
            .orElse(null);
`

Reply:
will do.